src/main/java/com/thullo/service/BoardServiceImpl.java

```diff
@@ -79,24 +79,24 @@ public Board getBoardByTag(String boardTag) {
 
     public BoardResponse getBoardResponse(Board board) {
         BoardResponse boardResponse = mapper.map(board, BoardResponse.class);
-
         List<Task> tasks = board.getTasks();
+        Set<String> status = new LinkedHashSet<>(Arrays.asList("BACKLOG", "IN_PROGRESS", "IN_REVIEW", "COMPLETED"));
```

Comment:
**NOTE**, If we agree to replace spaces with underscores and capitalize status, it's important that we remain consistent. To ensure consistency, we will create a helper function that can take a status and convert it to the proper format.

It might not be in this implementation though